README.md

```diff
@@ -20,6 +20,7 @@ MCAP libraries are provided in the following languages. For guidance on each lan
 | Python                | [readme](./python)     | [API docs](https://mcap.dev/docs/python)                        | `mcap`       | [![](https://shields.io/pypi/v/mcap)](https://pypi.org/project/mcap/)                |
 | JavaScript/TypeScript | [readme](./typescript) | [API docs](https://mcap.dev/docs/typescript)                    | `@mcap/core` | [![](https://shields.io/npm/v/@mcap/core)](https://www.npmjs.com/package/@mcap/core) |
 | Swift                 | [readme](./swift)      | [API docs](https://mcap.dev/docs/swift/documentation/mcap)      |              | see [releases](https://github.com/foxglove/mcap/releases)                            |
+| Rust                  |                        | [API docs](https://mcap.dev/docs/rust/doc/mcap)                 | `mcap`       | [![](https://shields.io/crates/v/mcap)](https://crates.io/crate/mcap)                |
```

Comment:
I suggest adding a rust readme and explaining the current features/limitations in it.